detox/local-cli/detox-build.js

```diff
@@ -27,3 +29,15 @@ if (buildScript) {
 } else {
   throw new Error(`Could not find build script in detox.configurations["${program.configuration}"]`);
 }
+
+function getConfigurationFile(configPath) {
+  let config;
+  if (configPath) config = require(path.join(process.cwd(), configPath));
```

Comment:
What would happen if configPath is not a relative path, say like `/Users/me/myConfig.json`?